rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -353,13 +364,13 @@ def __next__(self):
 
             self.output_tensor_list[i].copy_data(ctypes.c_void_p(output.data_ptr()), self.output_memory_type)
             self.output_list.append(output)
-
-
-        self.labels = self.loader.get_image_labels()
-        self.labels_tensor = self.labels_tensor.copy_(torch.from_numpy(self.labels)).long()
-        return self.output_list, self.labels_tensor
```

Comment:
Please check if this code deletion does not create any issues and handles all cases as previously